napalm/base/helpers.py

```diff
@@ -258,52 +258,57 @@ def as_number(as_number_val):
         return int(as_number_str)
 
 
-def int_split_on_match(split_interface):
-    '''
-    simple fuction to split on first digit, slash, or space match
-    '''
-    head = split_interface.rstrip(r'/\0123456789 ')
-    tail = split_interface[len(head):].lstrip()
-    return head, tail
-
-
-def canonical_interface_name(interface, update_os_mapping=None):
-    '''
-    Function to retun interface canonical name
+def split_interface(intf_name):
+    """Split an interface name based on first digit, slash, or space match."""
+    head = intf_name.rstrip(r'/\0123456789. ')
+    tail = intf_name[len(head):].lstrip()
+    return (head, tail)
+
+
+def canonical_interface_name(interface, addl_name_map=None):
```

Comment:
What is `addl_name_map`?

Reply:
A map to add or override in case there is vendor specific info

Reply:
Ok, I added a comment on the other method but it'd be great to format the docstring properly explaining the arguments and adding it to the documentation with autodoc.

Reply:
Will do